src/xcode/ENA/ENA/Source/Scenes/Home/Cells/Statistics/HomeStatisticsTableViewCell.swift

```diff
@@ -113,46 +167,71 @@ class HomeStatisticsTableViewCell: UITableViewCell {
 				Log.debug("update with \(keyFigureCellModel.selectedLocalStatistics.count) local stats", log: .localStatistics)
 				
 				self.removeLocalStatisticsCards()
-
+				
 				for singleSelectedLocalStatistics in selectedLocalStatistics {
+					
+					// check to separate the single entry and multiple entries
+					self.localStatisticsRegion = singleSelectedLocalStatistics.localStatisticsRegion
+					let regionName = singleSelectedLocalStatistics.localStatisticsRegion.name
+					let regionStatistics: SevenDayData
```

Comment:
Also a possible name: `RegionStatistics` or `RegionStatisticsData`

Reply:
you mean instead of SevenDayData?